dev/ci/user-overlays/08779-ppedrot-rm-implicit-tactic.sh

```diff
@@ -0,0 +1,8 @@
+_OVERLAY_BRANCH=rm-implicit-tactic
+
+if [ "$CI_PULL_REQUEST" = "8779" ] || [ "$CI_BRANCH" = "$_OVERLAY_BRANCH" ]; then
+
+    ltac2_CI_BRANCH="$_OVERLAY_BRANCH"
+    ltac2_CI_REF=https://github.com/ppedrot/ltac2
```

Comment:
should be CI_REF=$branch and CI_GITURL=https://...

Reply:
Arf, stupid me. Let's fix it.